nautobot/extras/filters/customfields.py

```diff
@@ -70,7 +70,15 @@ class CustomFieldJSONFilter(CustomFieldFilterMixin, django_filters.Filter):
     """Custom field single value filter for backwards compatibility"""
 
 
-class CustomFieldMultiSelectFilter(CustomFieldFilterMixin, django_filters.Filter):
+class CustomFieldSelectFilter(CustomFieldFilterMixin, django_filters.Filter):
```

Comment:
Do we actually need these two filter classes, or should we just use a `CustomFieldMultiValueCharFilter` for select/multiselect custom fields?

Reply:
Because of the `lookup_expr`, Yes we would need both, `Select` makes use of the `in` lookup expr and `MultiSelect` `contains`

Reply:
Should these be subclasses of `CustomFieldMultiValueCharFilter`?